contracts/contracts/sbtc-deposit.clar

```diff
@@ -2,10 +2,12 @@
 
 ;; constants
 (define-constant txid-length u32)
+(define-constant dust-limit u100)
```

Comment:
Dust limit is 546, unless you intended for this not to match the BTC dust limit

Reply:
Gotcha, thanks for the reminder, addressed.